packages/vite/package.json

```diff
@@ -13,7 +13,10 @@
   "types": "./dist/node/index.d.ts",
   "exports": {
     ".": {
-      "types": "./dist/node/index.d.ts",
+      "types": {
+        "import": "./dist/node/index.d.ts",
+        "require": "./index.d.cts"
```

Comment:
Could this file live in `./dist/node/index.d.cts` ? And be autogenerated during build to avoid the two linting issues?

Reply:
Just pointing to `./dist/node-cjs/index.d.cts` should work, actually